Tools/GenerateWebsiteData.ps1

```diff
@@ -77,6 +79,58 @@ function GetLinks ($docObj) {
     return $null
 }
 
+# Generate a PSObject that will be used to populate the cmdlet JSON file based on $parameter.
+function New-DynamicParamObject($parameter) {
+    if ($parameter.Position -ge 0) {
```

Comment:
This is a matter of style, but personally I prefer to declare the variable with a default value and then change it after the fact. Rather than set its value inside the if-body, and then use it outside of the if body. Because the way it is used now it isn't 100% clear where $position is declared. (Since it is never declared, just assigned to.)